charts/generic/k8-custom-metrics-apiserver/experiment.yaml

```diff
@@ -0,0 +1,58 @@
+# Generic Chaos experiment for Application team, who want to participate in Game Day
```

Comment:
@dharmaanu , this is a custom solution and don't have kubernetes resource associate, we don't need this chaos-chart.